src/applications/static-pages/subscribeAccordionEvents.js

```diff
@@ -0,0 +1,15 @@
+import recordEvent from 'platform/monitoring/record-event';
+
+export default function subscribeAccordionEvents() {
+  document.body.addEventListener(
+    '@department-of-veterans-affairs/formation/accordion/button-clicked',
+    e => {
+      recordEvent({
+        event:
+          e.detail.toggle === 'expand'
+            ? 'nav-accordion-expand'
+            : 'nav-accordion-collapse',
+      });
```

Comment:
Picky, but maybe we could do `nav-accordion-${e.detail.toggle}`?

Reply:
Update pushed.  Thx!

Reply:
Re. the tools repo PR154, the formation version-bump is already included.

Once approved, I'll merge and publish, then I'll push a formation bump here to match.